ipaserver/rpcserver.py

```diff
@@ -832,6 +832,18 @@ class login_kerberos(KerberosLogin):
 class login_x509(KerberosLogin):
     key = '/session/login_x509'
 
+    def __call__(self, environ, start_response):
+        self.debug('WSGI login_x509.__call__:')
+
+        try:
```

Comment:
I assume that this operation called always, so from performance POW it should be faster. Please use:

```python
if 'KRB5CCNAME' not in environ:
     return ...
```

Exception handling is expensive.
But we are talking about miliseconds so I'm fine with this too